BLE_BatteryLevel/source/pretty_printer.h

```diff
@@ -65,7 +65,7 @@ inline void print_error(ble_error_t error, const char* msg)
 }
 
 /** print device address to the terminal */
-inline void print_address(const Gap::Address_t &addr)
+inline void print_address(const BLEProtocol::AddressBytes_t &addr)
```

Comment:
Ok, for this PR, latter this will have to replace with the up to date types.

Reply:
Yea we probably need another example update after Gap refactoring is done